### PR TITLE
fix(integrations & plugins): [sc-8971] Add nonce to Magento 2 inline script

### PR DESCRIPTION
**Pull request by Jamstaz**
Story details: https://app.shortcut.com/salesfire/story/8971

### CODE REVIEW

**Review by JackGreen411 — DISMISSED** [review]

**Review by brandonkyaw — DISMISSED** [review]

File: Block/Script.php [hunk 1]
```diff
@@ -149,6 +155,12 @@ public function _toHtml()
             ]));
         }
 
-        return $formatter->toScriptTag();
+        $nonce = null;
+        if (version_compare($this->getHelper()->getMagentoVersion(), '2.4.7', '>=')) {
```
Comment (elliotlings) [inline Block/Script.php]: Do you think we could make a method for this to make it easier to use in the future? `$this->getHelper()->isMinimumMagentoVersion('2.4.7');`?

**Review by elliotlings — DISMISSED** [review]

**Comment by elliotlings** [comment]
We need to bump all the version numbers, too, as noted in the SUBMITTING.md.

File: Block/Script.php (hunk 1 above)
Comment (Jamstaz) [reply Block/Script.php]: Yeah I think that's a good idea. I've added that function.

**Comment by Jamstaz** [comment]
> We need to bump all the version numbers, too, as noted in the SUBMITTING.md.

Version has been bumped to v1.4.11.

**Review by elliotlings — DISMISSED** [review]

**Review by elliotlings — APPROVED** [review]

**Review by JackGreen411 — APPROVED** [review]

**Pull request merged**